ui/src/app/views/workspace/component.html

```diff
@@ -107,8 +107,18 @@ <h3 class="modal-title">Cluster Created:</h3>
   <div class="area-block">
     <div class="table-header">
       <h3 class="table-header-descriptor">Notebooks</h3>
+      <div class="alert {{alertCategory}}" *ngIf="showAlerts">
+        <div class="alert-items">
+          <div class="alert-item static">
+            <div class="alert-icon-wrapper">
+              <clr-icon class="alert-icon" shape="check-circle"></clr-icon>
+            </div>
+            <span class="alert-text">{{alertMsg}}</span>
+          </div>
+        </div>
+      </div>
       <button class="btn" (click)="launchNotebook()">Launch Notebook Server</button>
-      <button class="btn" (click)="createAndLaunchNotebook()">
+      <button class="btn" id="createAndLaunch" (click)="createAndLaunchNotebook()">
```

Comment:
If possible, I would do this as a class rather than an id, because id's need to be globally unique. @dmohs has just taught me to be very wary of using IDs anywhere.